io/js/src/main/scala/fs2/io/net/NetException.scala

```diff
@@ -107,4 +108,8 @@ object UnknownHostException {
       case _ => None
     }
   private[this] val pattern = raw"(?:ENOTFOUND|EAI_AGAIN)(?: (\S+))?".r
+  private[net] val message = osMod.`type`() match {
```

Comment:
What about Windows?

Reply:
If someone can run on Windows and tell me what JVM does I'm happy to add it!